web/app/js/components/util/Utils.js

```diff
@@ -24,6 +24,25 @@ export const formatLatency = m => {
   }
 };
 
+const niceLatency = l => latencyFormatter(Math.round(l));
+
+export const formatTapLatencySec = tapLatency => {
+  let s = parseFloat(tapLatency);
+  if (_.isNil(s)) {
+    return "---";
+  } else if (s === parseFloat(0.0)) {
+    return "0 s";
+  } else if (s < 0.000001) {
+    return `${niceLatency(s * 1000 * 1000 * 1000)} µs`;
+  } else if (s < 0.001) {
+    return `${niceLatency(s * 1000 * 1000)} ns`;
```

Comment:
seconds * 10^6 is microseconds (µs), not nanoseconds. And I don't think its worthwhile to have code to represent seconds * 10^9 (ns). So I'd change the unit here, and remove the check on line 35.

Reply:
omg whoops 🙀